src/components/OrgView/ActivityButton/ActivityButton.js

```diff
@@ -0,0 +1,137 @@
+import React, { useCallback, useContext, useRef, useState } from 'react'
+import PropTypes from 'prop-types'
+import { Spring, animated } from 'react-spring'
+import {
+  ButtonIcon,
+  IconAlert,
+  Popover,
+  GU,
+  Tag,
+  springs,
+  useTheme,
+} from '@aragon/ui'
+import ActivityList from '../../../components/Activity/ActivityList'
+import { ActivityContext } from '../../../contexts/ActivityContext'
+import { AppType } from '../../../prop-types'
+
+const { div: AnimDiv } = animated
+
+const ActivityButton = React.memo(function ActivityButton({ apps }) {
+  const theme = useTheme()
+  const [opened, setOpened] = useState(false)
+  const { markActivitiesRead, unreadActivityCount } = useContext(
+    ActivityContext
+  )
+  const containerRef = useRef()
+
+  const handleToggle = useCallback(
+    () =>
+      setOpened(opened => {
+        if (opened) {
+          markActivitiesRead()
+        }
+        return !opened
+      }),
+    [markActivitiesRead]
+  )
+  const handleClose = useCallback(() => {
+    markActivitiesRead()
+    setOpened(false)
+  }, [markActivitiesRead])
+
+  return (
+    <React.Fragment>
+      <div ref={containerRef} css="width: 58px">
+        <ButtonIcon
+          element="div"
+          onClick={handleToggle}
+          css={`
+            height: 100%;
+            width: 100%;
+            border-radius: 0;
+            padding-left: ${0.5 * GU}px;
+            justify-content: space-between;
+
+            /* This is a bit of a hack to get the focus ring to appear only
+             * around the button and not the spacer
+             */
+            &:focus:after {
+              right: ${3 * GU}px;
```

Comment:
@bpierre This is a bit of a brittle hack but I didn't want to change `ButtonBase`'s `focusRingSpacing` API to get too complicated.